packages/plugin-ext/src/main/browser/keybindings/keybindings-contribution-handler.ts

```diff
@@ -75,7 +75,7 @@ export class KeybindingsContributionPointHandler {
 
     private handlePartialKeybindings(keybinding: Keybinding, partialKeybindings: Keybinding[]) {
         partialKeybindings.forEach(partial => {
-            if (keybinding.context === undefined || keybinding.context === partial.context) {
```

Comment:
Should not we still respect `context` as well? It is not deprecated yet.

Reply:
If I understand correctly, I believe that the original keybinding.context was always undefined. [this](https://github.com/theia-ide/theia/blob/master/packages/plugin-ext/src/main/browser/keybindings/keybindings-contribution-handler.ts#L38) converts a plugin keybinding to a theia keybinding and it does not have a context in the [object when it returns](https://github.com/theia-ide/theia/blob/master/packages/plugin-ext/src/main/browser/keybindings/keybindings-contribution-handler.ts#L60).

Reply:
True, in plugin system it is always undefined